lib/Conversion/HWToBTOR2/HWToBTOR2.cpp

```diff
@@ -474,10 +489,12 @@ struct ConvertHWToBTOR2Pass
       width = hw::getBitWidth(reg.getType());
       next = reg.getInput();
       resetVal = reg.getResetValue();
+
```

Comment:
Nit: Blank lines before `else` are rather unusual style, but apparently clang-format does not mind...